cmd/podman/pod_create.go

```diff
@@ -11,6 +12,11 @@ import (
 	"github.com/urfave/cli"
 )
 
+var (
+	// CRI-O default kernel namespaces
+	DefaultKernelNamespaces = "ipc,net,pid"
```

Comment:
I don't think we want to share PID by default - fairly certain CRI-O doesn't?

We might want to share UTS though.

Reply:
@mrunalp @rhatdan WDYT?

Reply:
Pid is not shared by default at least originally.  So I don't think we should be doing it either.

Reply:
http://cri-o.io/ Mentions "Pods are a kubernetes concept consisting of one or more containers sharing the same IPC, NET and PID namespaces and living in the same cgroup." Totally open to leaving it as ipc,net,uts but thought I'd throw it out there @rhatdan @mheon